client/src/shared/ui/overlay/Overlay.less

```diff
@@ -84,8 +84,10 @@
   
   fieldset {
     border: none;
-    margin: 0 0 5px 0;
-    padding: 0;
+    margin: 0 -12px 5px 0;
+    padding: 0 12px 0 0;
+    max-height: calc(100vh - 176px);
+    overflow-y: scroll;
```

Comment:
On my machine, the scrollbar is visible even though it isn't necessary

![image](https://user-images.githubusercontent.com/9433996/163969631-b5bea194-4735-44c0-96f0-185842ba2e1f.png)

I think that can be fixed via `overflow-y: auto`.

```suggestion
    overflow-y: auto;
```

Reply:
Ah that doesn't happen on mine. But I will switch to this.